api/task_processor/management/commands/runprocessor.py

```diff
@@ -21,7 +22,7 @@
 
 logger = logging.getLogger(__name__)
 
-TASKS_MODULES_TO_RELOAD = [processor_tasks, sse_tasks]
+TASKS_MODULES_TO_RELOAD = [processor_tasks, sse_tasks, auth_tasks]
```

Comment:
I had to add auth_tasks here because that module is not imported by anything, i.e: python will not load it otherwise

Reply:
It's considered good practice to use `AppConfig.ready()` for such cases, e.g.

```python
# custom_auth/apps.py

class CustomAuthAppConfig(AppConfig):
    name = "custom_auth"
   
    def ready(self) -> None:
        from custom_auth import tasks
```

```python
# settings/common.py

...

INSTALLED_APPS = [
    ...
    "custom_auth",
    ...
]
```

Reply:
I guess the phrased the comment wrong, we need a way to load it but not before we set `RUN_BY_PROCESSOR`. Importing tasks in ready will mean that we need to reload tasks here anyway

Reply:
I'd set it outside the command then, e.g. in the `run_task_processor` bash command in the entrypoint. Seems less hacky that way.

Reply:
Yeah, that's a great idea!

Reply:
Done